modules/ui_patterns_layouts/src/Plugin/Layout/PatternLayout.php

```diff
@@ -139,7 +151,7 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
         '#default_value' => $configuration['pattern']['variant'],
       ];
     }
-
+    $this->moduleHandler->alter('ui_patterns_layouts_form', $form['pattern'], $definition, $configuration);
```

Comment:
Can we please rename this hook into this?
```
ui_patterns_layouts_display_settings_form()
```
So we keep naming consistency with the other one. Thanks!